src/views/InvolvementProfile/index.jsx

```diff
@@ -325,10 +318,7 @@ const InvolvementProfile = () => {
                   <Cropper
                     ref={cropperRef}
                     src={preview}
-                    style={{
-                      maxWidth: maxCropPreviewWidth(),
-                      maxHeight: maxCropPreviewWidth() / cropperData.aspectRatio,
-                    }}
+                    className={styles._cropper / cropperData.aspectRatio}
```

Comment:
here they were dividing two numbers, I think this is a case where we'll have to leave the style tag, since they are using functions and variables that they compute in the code (maxCropPreviewWidth() and cropperData.aspectRatio are both Javascript, not usable in CSS)